web/ui/react-app/src/pages/Flags.tsx

```diff
@@ -1,6 +1,52 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import { RouteComponentProps } from '@reach/router';
+import { Alert, Table } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
 
-const Flags: FC<RouteComponentProps> = () => <div>Flags page</div>;
+const Flags: FC<RouteComponentProps> = () => {
+  const [flags, setFlags] = useState<{ [key: string]: string } | null>(null);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    fetch('../api/v1/status/flags')
+      .then(res => res.json())
+      .then(res => setFlags(res.data))
+      .catch(error => setError(error.message));
+  }, []);
+
+  const body = (error: string, flags: { [key: string]: any } | null) => {
```

Comment:
It looks like the type here is slightly different from the type of the `flag` state